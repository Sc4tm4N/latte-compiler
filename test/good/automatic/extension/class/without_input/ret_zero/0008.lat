int c(Node node) {
   int previous = node.a;
   node.a = 150;
   return previous;
}

class Node {
  Node next;
  int a;
}

int main() {
  Node node = new Node;
  printInt(node.a);
  node.a = 10;
  printInt(node.a);
  node.a++;
  printInt(node.a);
  printInt(c(node));
  printInt(node.a);
  return 0;
}
