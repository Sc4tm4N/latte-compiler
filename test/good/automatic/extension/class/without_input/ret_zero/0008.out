0
10
11
11
150
